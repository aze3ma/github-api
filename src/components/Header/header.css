.header {
    padding: 1rem;
}

.header__logo {
    display: flex;
    font-size: 1.2rem;
    font-weight: bold;
}
.header__logo span {
    font-size: 1.2rem;
    font-weight: bold;
}
.header__logo img {
    max-width: 40px;
    margin-right: 1rem;
    object-fit: contain;
}

.header__caption {
    font-size: 1rem;
    color: var(--grey-300);
    position: absolute;
    top: 2.5rem;
    left: 15rem;
}
