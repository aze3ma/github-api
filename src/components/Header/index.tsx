import React from 'react'

import './header.css'

const Header = () => (
    <header className="header">
        <h1 className="header__logo">
            <img src="/images/logo.png" alt="Github search" />
            <span>Github Searcher</span>
        </h1>
        <p className="header__caption">Search users or repositories</p>
    </header>
)

export default Header
