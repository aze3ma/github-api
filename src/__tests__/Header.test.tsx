import React from 'react'

import { render } from '@testing-library/react'

import Header from '../components/Header'

describe('Header.tsx', () => {
    it('should render correctly', () => {
        const { container } = render(<Header />)
        expect(container).toMatchSnapshot()
    })
})
