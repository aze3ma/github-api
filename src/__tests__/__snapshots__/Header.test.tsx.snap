// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.tsx should render correctly 1`] = `
<div>
  <header
    class="header"
  >
    <h1
      class="header__logo"
    >
      <img
        alt="Github search"
        src="/images/logo.png"
      />
      <span>
        Github Searcher
      </span>
    </h1>
    <p
      class="header__caption"
    >
      Search users or repositories
    </p>
  </header>
</div>
`;
